_includes/header.html

```diff
@@ -0,0 +1,42 @@
+<header>
+<nav class="navbar navbar-expand-lg navbar-light" style="background-color: #8000ff;">
+    <a class="navbar-brand" href="#">
+        <img src="assets\img\A6 Games Logo.png" width="30" height="30" class="d-inline-block align-top" alt=""> 
```

Comment:
This wasn't in the ticket but let's se the width="auto" this will ensure we maintain the aspect ratio of whatever the image is.
```suggestion
        <img src="assets\img\A6 Games Logo.png" width="auto" height="30" class="d-inline-block align-top" alt=""> 
```